dlt/destinations/impl/snowflake/configuration.py

```diff
@@ -49,6 +49,9 @@ def _read_private_key(private_key: str, password: Optional[str] = None) -> bytes
     )
 
 
+snowflake_application_id = "dltHub_dlt"
```

Comment:
is this the application id we picked for `dlt`? @VioletM ?

Reply:
I would make the var uppercase and import it into the test so we don't have any magic strings there

Reply:
Done.